src/Umbraco.Core/Auditing/IdentityAuditEventArgs.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Threading;
+using System.Web;
+using Umbraco.Core.Security;
+
+namespace Umbraco.Core.Auditing
+{
+    /// <summary>
+    /// This class is used by events raised from hthe BackofficeUserManager
+    /// </summary>
+    public class IdentityAuditEventArgs : EventArgs
```

Comment:
Problem with this class is that most of these properties are mutable which means that if multiple event handlers subscribe to the event, they can change all of these properties which will then be changed for other handlers of the event. 

These properties should be immutable (readonly) and set via the ctor(s) of the class

Reply:
I've pushed a fix for this